core/storage/suggestion/gae_models.py

```diff
@@ -665,6 +665,93 @@ def get_in_review_translation_suggestions_by_offset(
             next_offset
         )
 
+    @classmethod
+    def get_reviewable_translation_suggestions_for_single_exploration(
+        cls,
+        offset: int,
```

Comment:
Do we still need an offset here if the intention is to return all the results?

Reply:
No, I have removed it from here, but I am returning offset to ensure consistency among all return types.